cpp/doxygen/developer_guide/BENCHMARKING.md

```diff
@@ -54,3 +54,31 @@ reaches its saturation bottleneck, whether that bottleneck is bandwidth or compu
 sets larger than this point is generally not helpful, except in specific cases where doing so
 exercises different code and can therefore uncover regressions that smaller benchmarks will not
 (this should be rare).
+
+## Running and Comparing NVBench Benchmarks in libcudf
+
+### Running Benchmarks
+
+By default, benchmarks are **not** built as part of the libcudf build process. To enable them, pass the following flag to CMake:
+
+```bash
+cmake -DBUILD_BENCHMARKS=ON /path/to/cpp/build/latest/
+```
+This will build the NVBench benchmark executables under the `cpp/build/latest/benchmarks`
+directory. Each benchmark is compiled into its own binary with a `_NVBENCH` suffix.
+
+To list available benchmarks:
+
+```bash
+ls cpp/build/latest/benchmarks/*_NVBENCH
+```
+
+To view benchmark options:
+
+```bash
+./cpp/build/latest/benchmarks/RESHAPE_NVBENCH --help
+```
```

Comment:
```suggestion
To view benchmark options, use `--help` argument such as:

```bash
./cpp/build/latest/benchmarks/RESHAPE_NVBENCH --help
```
```

Reply:
Or this one works too

```suggestion
To view benchmark options:

```bash
./cpp/build/latest/benchmarks/<NAME>_NVBENCH --help
```
```